mapclientplugins/geometryfitter/view/geometryfitterwidget.py

```diff
@@ -410,6 +423,8 @@ def _updateDisplayWidgets(self):
         self._ui.displaySurfacesExterior_checkBox.setChecked(self._model.isDisplaySurfacesExterior())
         self._ui.displaySurfacesTranslucent_checkBox.setChecked(self._model.isDisplaySurfacesTranslucent())
         self._ui.displaySurfacesWireframe_checkBox.setChecked(self._model.isDisplaySurfacesWireframe())
+        self._ui.displayRMSError_lineEdit.setText("TEST!")
+        self._ui.displayMaxError_lineEdit.setText("TEST22!")
```

Comment:
These "values" will come from the model in the future `self._model.rmsError()` and `self._model.maxError()`. Might as well have these in place now.